torchvision/prototype/transforms/_misc.py

```diff
@@ -79,3 +81,28 @@ def __init__(self, dtype: torch.dtype, *types: Type) -> None:
 
     def extra_repr(self) -> str:
         return ", ".join([f"dtype={self.dtype}", f"types={[type.__name__ for type in self.types]}"])
+
+
+class CleanupBoxes(Transform):
+    def _get_params(self, sample: Any) -> Dict[str, Any]:
+        bounding_boxes = query_bounding_box(sample)
+        bounding_boxes_clamped = F.clamp_bounding_box(
```

Comment:
Shouldn't we be doing this clamp in the transform to ensure all values are valid? Without this clamp, we might have x1,y1 values that are negative on the final boxes. Or we might end up with x2,y2 that are beyond the size of the image.